addons/mail/static/src/models/messaging/messaging.js

```diff
@@ -56,6 +56,16 @@ function factory(dependencies) {
             }
         }
 
+        async getGroupChat(partnerIds) {
+            const chat = await this.async(() =>
+                this.env.models['mail.thread'].performRpcCreateGroupChat({
+                    partnerIds: partnerIds,
+                })
+            );
+
+            return chat;
+        }
```

Comment:
Not really a get, but a create group chat.
Also prefer using records than fields of record (here: instead of using `partnerIds`, use `partners`).

Reply:
Sorry I wasn't clear enough: I didn't mean to pass records to the RPC, as client/server communication do not support passing records (hence they share ids instead).

In the client-side code, when you call a function and some arguments refer to records, use records instead of ids.
Here, this means use `partners` as param of both `getGroupChat` (now `createGroupChat`) and `performRpcCreateGroupChat`. It's best to refer to records by passing records rather than fields of records that identifies them.